behavior_tests/src/bt-code-pin-log-filename/test.cu

```diff
@@ -0,0 +1,50 @@
+//==============================================================
+// Copyright 2019 Intel Corporation
```

Comment:
```suggestion
// Copyright (C) Intel Corporation
```

Reply:
@ShengchenJ This issue I will fix in other PR, could you please merge it first? For feature board update.

Reply:
Please update it in a new PR, thanks.